### ISSUE TITLE
Unable to edit profile picture by choosing image from photos / library

### ISSUE THREAD

**Issue by tomglanville** [jiscdev/study-goal#102]
Error when changing profile picture using select from library / photos option. Appears to timeout on the request after about 10.

Error log below:

```
> W/System.err: java.io.FileNotFoundException: http://stuapp.analytics.alpha.jisc.ac.uk/fn_edit_profile_picture
> W/System.err:     at com.android.okhttp.internal.http.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:197)
> W/System.err:     at com.studygoal.jisc.Managers.NetworkManager$uploadProfileImage.call(NetworkManager.java:365)
> W/System.err:     at com.studygoal.jisc.Managers.NetworkManager$uploadProfileImage.call(NetworkManager.java:294)
> W/System.err:     at java.util.concurrent.FutureTask.run(FutureTask.java:237)
> W/System.err:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
> W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
> W/System.err:     at java.lang.Thread.run(Thread.java:818)
> 
```

Might be a back-end issue, if so will pass onto our team. Thanks

**Comment by mparker-tbox** [comment]
Should be resolved

**Comment by tomglanville** [comment]
Fixed - will be part of v1.0.3.
